functionaltests/src/test/java/util/constant/WebDriverCommon.java

```diff
@@ -5,7 +5,9 @@
  */
 public final class WebDriverCommon {
 
-    public static final String XPATH_SUBMIT = "//button[@type='submit']";
+    //public static final String XPATH_SUBMIT = "//button[@type='submit']";
+    public static final String CLASS_NAME_PRIMARY = "primary";
```

Comment:
What about `SUBMIT_BTN_CLASS_NAME`?

Reply:
it is not a submit button any more though

Reply:
But you use it in `findSubmitButton()`?

Reply:
accept it as an abstraction: findSubmitButton() search for a button which will submit the values to the server. Though, this might be (and actually it is not any more) an `<input type="submit">`, looks like they don't use HTML forms any more